### ISSUE TITLE
Could not find litho:yogajni:unspecified.

### ISSUE THREAD

**Issue by amoikevin** [facebook/litho#612]
- [ ] I have searched [existing issues](https://github.com/facebook/litho/issues) and this is not a duplicate

## Version
0.32.0

## Issues and Steps to Reproduce
Could not find litho:yogajni:unspecified.
Required by:
    project :police > com.facebook.litho:litho-core:0.32.0

## Link to Code
implementation 'com.facebook.litho:litho-core:0.32.0'

**Comment by bangonkali** [comment]
I have the same issue:

```
    implementation 'com.facebook.litho:litho-core:0.32.0'
    implementation 'com.facebook.litho:litho-widget:0.32.0'
    kapt 'com.facebook.litho:litho-processor:0.32.0'
    implementation 'com.facebook.soloader:soloader:0.5.1'
    implementation 'com.facebook.litho:litho-fresco:0.32.0'
    testImplementation 'com.facebook.litho:litho-testing:0.32.0'
    implementation 'com.facebook.litho:litho-sections-core:0.32.0'
    implementation 'com.facebook.litho:litho-sections-core:0.32.0'
    implementation 'com.facebook.litho:litho-sections-widget:0.32.0'
    compileOnly 'com.facebook.litho:litho-sections-annotations:0.32.0'
    kapt 'com.facebook.litho:litho-sections-processor:0.32.0'
```

```
Android Studio 4.0 Canary 3
Build #AI-192.6817.14.36.5994236, built on November 8, 2019
Runtime version: 1.8.0_212-release-1586-b4-5784211 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.15.1
GC: ParNew, ConcurrentMarkSweep
Memory: 8125M
Cores: 8
Registry: ide.new.welcome.screen.force=true
Non-Bundled Plugins: 
```

Does anyone have any idea how to work around this?

**Comment by passy** [comment]
Thanks for the report. Something went wrong with the release and we'll put up a patch release to remedy this Soon™️.

**Comment by mihaelao** [comment]
Should be fixed now - if still seeing issues you probably need to clean the gradle cache.

**Comment by muraziz** [comment]
Hey @amoikevin, sorry for that, we re-published 0.32.0 with correct linking to yoga lib, so it should work now :)

**Comment by mfatiga** [comment]
This does not work still on litho 0.50.0
